src/javascript/app/common/account_opening.js

```diff
@@ -124,6 +125,7 @@ const AccountOpening = (() => {
                 }
             }
             $address_state.parent().parent().setVisibility(1);
+            Dropdown('#address_state');
```

Comment:
from the function name it's not very intuitive what it's supposed to do, can we change to something like `createDropdown`, or maybe `transformDropdown` even